frontend_lib/src/appContentFactory.js

```diff
@@ -143,8 +145,15 @@ export function appContentFactory (WrappedComponent) {
       // see https://github.com/tracim/tracim/issues/1101
       const newCommentForApi = isCommentWysiwyg ? newComment : Autolinker.link(`<p>${convertBackslashNToBr(newComment)}</p>`)
 
+      let newCommentForApiWithMention
```

Comment:
If you went with my suggestion for the variable name in my earlier comment, you might want to change it here too.